frontend/package.json

```diff
@@ -38,7 +38,8 @@
     "redux-logger": "^3.0.6",
     "redux-mock-store": "^1.5.3",
     "redux-thunk": "^2.2.0",
-    "reselect": "^4.0.0"
+    "reselect": "^4.0.0",
+    "tus-js-client": "^1.6.0"
```

Comment:
We'll want someone with browserstack access to test this build, given that new dependencies are added. @FreshGyan @thegentlemanphysicist

Reply:
Agreed. We also have another dependency that was added through PR#496 for the file downloading.